routing/src/cpi.rs

```diff
@@ -182,11 +181,13 @@ pub fn start_cpi(
                         if event.is_writable() {
                             cached_sock.flush_out_fast();
                             if !cached_sock.interests().is_writable() {
-                                poller.registry().reregister(
+                                if let Err(e) = poller.registry().reregister(
                                     &mut SourceFd(&cached_sock.get_raw_fd()),
                                     CPSOCK,
                                     cached_sock.interests(),
-                                );
+                                ) {
+                                    error!("Too bad: poller registry failed: {e}");
```

Comment:
Approved on my side, providing you can just change this bit:

```suggestion
                                    error!("Poller registry failed: {e}");
```